test/react-native-cli/features/steps/steps.rb

```diff
@@ -69,6 +69,12 @@
   )
 end
 
+When('I build the Android app') do
+  # Handle both Dockerized and local Maze Runner executions
+  path = (File.exist? 'scripts/react-native-cli-helper.js') ? '.' : '../..'
+  `node -e 'require("#{path}/scripts/react-native-cli-helper").buildAndroid("./features/fixtures", "./local-build")'`
```

Comment:
Where is this relative to? `scripts` is four levels up locally so this is a bit confusing; I assume it works because the PWD would usually be `test/react-native-cli`?

I think this would be more resilient with an absolute path and check that the file exists in both cases so it's easier to debug if the file goes missing, e.g.

```suggestion
  # Handle both Dockerized and local Maze Runner executions

  possible_locations = [
    "#{__dir__}/../../scripts/react-native-cli-helper.js",
    "#{__dir__}/../../../../scripts/react-native-cli-helper.js",
  ]

  path = possible_locations.find { |path| File.exist?(path) }

  if path.nil?
    raise <<~ERROR
      The React Native helper script was not found in any of the expected locations:
        - #{possible_locations.join("\n  -")}
    ERROR
  end

  `node -e 'require("#{path}").buildAndroid("./features/fixtures", "./local-build")'` 
```

Reply:
Yeah, it was working due to PWD being as you say.  I've updated in line with your suggestion on my next set of changes (primarily for iOS) so we can get the code fix in ASAP.